test/verify/check-realms

```diff
@@ -311,6 +311,8 @@ class TestRealms(MachineCase):
         self.op_admin_password = "#realms-op-admin-password"
         self.domain_sel = "#system_information_domain_button"
         self.machine.execute("hostnamectl set-hostname x0.cockpit.lan")
+        # realm users sometimes have strange default locales, like en_US@POSIX.UTF-8
+        self.allow_journal_messages("invalid or unusable locale:.*")
```

Comment:
I just noticed that on my system (F32 toolbox) this now seems to affect *every* test, not just realms. So I'll re-drop this commit again.